templates/main.php

```diff
@@ -62,7 +62,8 @@
 		<?php print_unescaped($this->inc('part.navigation')); ?>
 		<?php /* print_unescaped($this->inc('part.settings')); */ ?>
 	</div>
-	<div id="app-content" ng-class="{ 'details-visible': sidebar.show }" ui-view>
+	<div id="app-content" ng-class="{ 'details-visible': sidebar.show }">
+		<div ui-view></div>
```

Comment:
the toggle button will be created by `js.js` from core (see line 1569) automatically. it works only if `ui-view` is in a separate div instead in `app-content`